cilium-sysdump/__main__.py

```diff
@@ -23,13 +23,31 @@
 import os
 import time
 import distutils.util
+import socket
+
 
 log = logging.getLogger(__name__)
 
 
+def is_ip_or_name(s):
+    if not s:
+        return False
+
+    if s[0].isdigit():
+        try:
+            socket.inet_aton(s)
```

Comment:
`socket.inet_aton` is IPv4 only. As we need to keep python2 support, please use `socket.inet_pton` with `socket.AF_INET` and `socket.AF_INET6` instead.

Reply:
Thanks! Fixed.